calendarsubscription/store.go

```diff
@@ -48,13 +49,13 @@ func NewStore(ctx context.Context, db *sql.DB, apiKeyring keyring.Keyring, oc on
 		authUser: p.P(`
 			UPDATE user_calendar_subscriptions
 			SET last_access = now()
-			WHERE NOT disabled AND id = $1
+			WHERE NOT disabled AND id = $1 AND created_at = $2
```

Comment:
I think `created_at` needs to be truncated before comparing -- the `IssuedAt` field only has a precision of seconds, but `TIMESTAMPTZ` is precise to microseconds. If I'm correct, it will be unlikely that the two will match during auth.
```suggestion
			WHERE NOT disabled AND id = $1 AND date_trunc('second', created_at) = $2
```